tensorflow/contrib/android/java/org/tensorflow/contrib/android/TensorFlowInferenceInterface.java

```diff
@@ -79,14 +176,14 @@ public TensorFlowInferenceInterface(AssetManager assetManager, String model) {
         throw new RuntimeException("Failed to load model from '" + model + "'", e);
       }
     }
-    
+
     try {
       if (VERSION.SDK_INT >= 18) {
         Trace.beginSection("initializeTensorFlow");
         Trace.beginSection("readGraphDef");
       }
-    
-      // TODO(ashankar): Can we somehow mmap the contents instead of copying them?
+
+     // TODO(ashankar): Can we somehow mmap the contents instead of copying them?
```

Comment:
alignment issue

Reply:
do you mind if I just delete it? since I am addressing it.